crates/oxc_linter/src/snapshots/empty_tags.snap

```diff
@@ -0,0 +1,75 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: empty_tags
+---
+  ⚠ eslint-plugin-jsdoc(empty-tags): Expects the void tags to be empty of any content.
+   ╭─[empty_tags.tsx:3:17]
+ 2 │                       /**
+ 3 │                        * @abstract extra text
+   ·                          ─────────
```

Comment:
The error message emphasizes "empty text" so I would put the span on the extra text.

Reply:
> https://github.com/oxc-project/oxc/pull/2815#issuecomment-2019347874

Let me reconsider this a bit...